src/client/components/Inputs/Select/Select.tsx

```diff
@@ -49,4 +49,9 @@ const Select: React.FC<SelectProps> = (props) => {
   )
 }
 
+Select.defaultProps = {
+  // eslint-disable-next-line react/default-props-match-prop-types
+  isClearable: true,
```

Comment:
It seems eslint has a problem with the optional 'isClearable' property from `Pick<ReactSelectProps, 'isClearable' | 'isMulti'> & ...`:

<img width="1079" alt="image" src="https://github.com/openforis/fra-platform/assets/41337901/99b65e78-0d0f-4a3b-8e05-6b93dfbdad85">

After trying some fixes, the problem seems to be that the type declaration is in another file and eslint doesn't record it.
One option to avoid this `eslint-disable-next-line` would be to make the prop non-optional and add `isClearable={true}` to all instances where the Select component is currently used (not ideal). I appreciate any suggestions.